sway/tree/view.c

```diff
@@ -437,7 +437,9 @@ void view_execute_criteria(struct sway_view *view) {
 		}
 		sway_log(SWAY_DEBUG, "for_window '%s' matches view %p, cmd: '%s'",
 				criteria->raw, view, criteria->cmdlist);
-		list_add(view->executed_criteria, criteria);
+		if (!criteria->continuous) {
```

Comment:
Here we might need some mechanism to prevent flapping, i.e. one continuous criterion matching and executing an action that makes another continuous criterion match and execute an action that in turn makes the first criterion match again and so on.